content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -184,18 +186,45 @@ otel:
     - b3
   resource:
     attributes:
-      environment: dev
-      xyz: foo
+      deployment.environment: dev
+      service:
+        name: cart
+        namespace: shop
 ```
 
 Environment variables example:
 
 ```shell
 export OTEL_PROPAGATORS="tracecontext,b3"
-export OTEL_RESOURCE_ATTRIBUTES="environment=dev,xyz=foo"
+export OTEL_RESOURCE_ATTRIBUTES="deployment.environment=dev,service.name=cart,service.namespace=shop"
 ```
 
-Disable the OpenTelemetry Starter:
+#### Overriding Resource Attributes
+
+As usual in Spring Boot, you can override properties in the
+`application.properties` and `application.yaml` files with environment
+variables.
+
+For example, you can set or override the `deployment.environment` resource
+attribute (not changing `service.name` or `service.namespace`) by setting the
+standard `OTEL_RESOURCE_ATTRIBUTES` environment variable:
+
+```shell
+export OTEL_RESOURCE_ATTRIBUTES="deployment.environment=prod"
+```
+
+Alternatively, you can use an environment variable to set or override a single
```

Comment:
```suggestion
Alternatively, you can use the
`OTEL_RESOURCE_ATTRIBUTES_DEPLOYMENT_ENVIRONMENT`
environment variable to set or override a single
```

Reply:
this is just an example, you could also use `OTEL_RESOURCE_ATTRIBUTES_SERVICE_NAMESPACE`

Reply:
Should that be the one we recommend?

Reply:
I think so, because it supports SpEL

Reply:
can this conversation be resolved to merge this PR?